composer.json

```diff
@@ -24,6 +24,7 @@
     "require": {
         "php": "^5.6 || ^7.0",
         "sonata-project/admin-bundle": "^3.28",
+        "sonata-project/block-bundle": "^3.11",
```

Comment:
If you ask me I do not like a lot having to add a dependency only to use a deprecated service that is only an extends from another class

Reply:
@jordisala1991 will it be ok, if i will increase version in `conflict` and `require-dev` sections and remove `block-bundle` from `require` instead of removing it?

Reply:
That could work better, not sure if we can apply this solution to all bundles not requiring block directly

Reply:
I understand your opinion @jordisala1991 but IMO the solution is straight forward and explicit

WDYT @greg0ire ?

Reply:
Well now that I think it, it is not a solution because with this change blockBundle becomes required.

Reply:
@OskarStark Also i could create CompilerPass, that replaces `sonata.templating` with `templating` if `sonata.templating` does not exists (this will be used for lower versions of BlockBundle). With new compiler pass changes in `composer.json` will be reverted.

This could be done for UserBundle, SeoBundle, MediaBundle, CommentBundle, ArticleBundle and NewsBundle

Reply:
@jordisala1991 @OskarStark Fixed. I added `sonata-project/block-bundle` to `conflict` and `require-dev` sections of composer.json